src/icons/ArrowIcon.js

```diff
@@ -4,7 +4,7 @@ import { Icon } from '../components/Icon'
 
 const ArrowIcon = (props) => (
   <Icon { ...props }>
-    <g transform='translate(0, -1)'>
+    <g transform='translate(0.75, -1)'>
```

Comment:
I think so, maybe my are not perfectly centered. I am not using those anymore. But they need to be perfectly centered from sketch I guess. hte rotate works probably from the center of the viwebox